src/libraries/System.Private.CoreLib/src/System/Diagnostics/StackTrace.cs

```diff
@@ -322,6 +324,16 @@ internal void ToString(TraceFormat traceFormat, StringBuilder sb)
                             sb.Append(' ');
                             sb.AppendFormat(CultureInfo.InvariantCulture, inFileLineNum, fileName, sf.GetFileLineNumber());
                         }
+                        else
+                        {
+                            if (s_ilOffsetToStackTrace)
+                            {
+                                sb.Append(' ');
+                                string methodToken = mb.MetadataToken.ToString("x");
+                                string ilOffset = sf.GetILOffset().ToString("x");
+                                sb.AppendFormat(CultureInfo.InvariantCulture, inFileILOffset, mb.Module.Name, methodToken, ilOffset);
```

Comment:
`Module.Name` will just return `<Unknown>` in single file mode, and in other cases where the module is not loaded from physical file on disk. Would it better to use Assembly simple name instead?

Reply:
As you said, assembly simple name looks better.